visidata/loaders/xml.py

```diff
@@ -46,7 +46,8 @@ def showColumnsBasedOnRow(self, row):
     def iterload(self):
         if isinstance(self.source, Path):
             from lxml import etree, objectify
-            self.root = etree.parse(self.source.open_text(encoding=self.options.encoding))
+            p = etree.XMLParser(huge_tree=True)
```

Comment:
The VisiData way to do this is:

```
p = etree.XMLParser(**self.options.getall('xml_'))
```

Then you can pass any of the possible kwarg options to XMLParser via standard VisiData options:

`options.xml_huge_tree = True`
`vd --xml-dtd-validation=True`

etc.

We might also want to make `huge_tree=True` the default, for the reasons you suggest, but also allow it to be re-set to False if someone didn't want it:

`vd.option('xml_huge_tree', True, 'explanation for what this option does')`